data/015/5008/en_US.json

```diff
@@ -3,11 +3,11 @@
 		"5008": {
 			"name": "Unknown",
 			"message": "You cannot post to Miiverse using\nthis Nintendo Network ID.\n\nFor details, please start Miiverse",
-			"short_description": "Your PNID has been banned from Juxtaposition",
-			"long_description": "Unknown cause.",
-			"short_solution": "Don't do whatever you did to get banned",
-			"long_solution": "Unknown solution.",
+			"short_description": "Your PNID has been restricted from Juxtaposition",
+			"long_description": "Your Pretendo Network ID has been restricted from Juxtaposition.\n\nThis typically occurs because of a violation of the Code of Conduct, or other offenses on the Network or the Discord server.",
```

Comment:
```suggestion
			"long_description": "Your Pretendo Network ID has been restricted on Juxtaposition.\n\nThis typically occurs because of a violation of the Code of Conduct, or other offenses on the Network or the Discord server.",
```